keymaps/markdown-preview.cson

```diff
@@ -1,4 +1,4 @@
-'atom-workspace, atom-workspace atom-text-editor':
+'.editor:not(.mini)':
```

Comment:
This should use the data attributes instead and keep targeting the element instead of the class.

```
atom-workspace atom-text-editor:not([mini])
```

See https://github.com/atom/atom/blob/74a627d41b41ce39e0a5ad43a08dbd4ce2a46fcc/keymaps/darwin.cson#L139 for an example